docs/hardhat.md

```diff
@@ -72,6 +72,33 @@ Once the correct endpoint is filled in the modal, just click on `OK` and the acc
 
 Now, one can start deploying the contract from Remix IDE to the Hardhat local node as usual.
 
+`console.log` in Remix IDE
+---------------------------
+
+_(Supported since Remix IDE v0.17.0)_
+
+Remix IDE supports hardhat console library while using `Remix VM`. It can be used while making a transaction or running unit tests.
+
+### Deploy and Run Transactions
+
+To try it out, you need to put an import statement and use `console.log` to print the value as shown in image.
+
+![](images/hardhat-console-file.png)
+
+Further, once you execute the `changeOwner` method, value from console statement will be shown in Remix terminal after transaction details as below:
```

Comment:
You need to add a "the":
Further, once you execute the `changeOwner` method, the value from ...

Reply:
updated